templates/types/streaming/fastapi/app/api/routers/chat.py

```diff
@@ -39,10 +38,7 @@ async def chat(
             f"Creating chat engine with filters: {str(filters)}",
         )
         chat_engine = get_chat_engine(filters=filters, params=params)
```

Comment:
how about creating the event_handler here?
```suggestion
       event_handler = EventCallbackHandler()
        chat_engine = get_chat_engine(filters=filters, params=params, event_handler=event_handler)
```